lib/parallel_tests/grouper.rb

```diff
@@ -103,7 +103,7 @@ def isolate_count(options)
       end
 
       def largest_first(files)
-        files.sort_by { |_item, size| size }.reverse
+        files.sort_by { |item, size| [size, item] }.reverse
```

Comment:
if it was sorted before it will stay sorted after, so this is the wrong spot to sort ?

Reply:
If I understand your comment correctly, you are saying that `files` is already sorted (as per your other comment), so `sort_by` size alone should be deterministic and the groups should be identical every time they are calculated, so there may be a better way to sort. Is my understanding right?

Reply:
yeah, this is not the place where things get shuffled, so it should not be there place where we sort